### ISSUE TITLE
Assignment of peer review for homework #5?

### ISSUE THREAD

**Issue by philstraforelli** [STAT545-UBC/Discussion#389]
Hello,

The deadline for peer-reviewing for homework #5 is today, but I haven't been assigned anyone to review yet - or maybe I accidentally deleted the messages that indicated my assignment. I just wanted a confirmation as to whether I missed my assignment, or everything is normal, or I've fallen through the cracks.

Thanks,

Phil

**Comment by jennybc** [comment]
@philstraforelli No, you are correct. I forgot to do this on Friday and now am thinking we will forego peer review on this last homework. People enrolled only in STAT 545 have every reason to expect that they are done with the course now.

Of course, you are welcome to visit other people's work or invite a specific peer to provide some feedback!

But unless I hear a great outpouring of grief, let's assume there will be no peer review for HW05.

**Comment by darwilliams** [comment]
Hooray! I was worried about this too.

**Comment by sinaneza** [comment]
Awesome!!!!!
